Extensions/servant.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE Servant **//
-//* VERSION 0.5.9 **//
+//* VERSION 0.6.0 **//
```

Comment:
Version number incorrectly incremented.  Should be +1 to `PATCH`, which would be `0.5.10`.